ledger/src/blockstore_options.rs

```diff
@@ -133,6 +133,18 @@ impl Default for ShredStorageType {
     }
 }
 
+impl ShredStorageType {
+    /// Returns ShredStorageType::RocksFifo where the specified
+    /// `shred_storage_size` is equally allocated to shred_data_cf_size
+    /// and shred_code_cf_size.
+    pub fn rocks_fifo(shred_storage_size: u64) -> ShredStorageType {
```

Comment:
Should the name here reflect that it yields a 1:1 ratio given that the default constants in `blockstore_db.rs` are not 1:1 (5 data : 4 coding).

Also since I took a look already, are the constants in `blockstore_db.rs` (DEFAULT_FIFO_COMPACTION_DATA_CF_SIZE & DEFAULT_FIFO_COMPACTION_CODING_CF_SIZE) ever used? I know they are referenced in `Default` trait implementation for `BlockstoreRocksFifoOptions`, but it looks like validator requires manual specification of the value. So, it seems that the `Default` trait & above mentioned constants are essentially unused, no?

Reply:
Yep, I think you're right.  The original one might be overly complicated, and it's in general okay to assume 1:1 between data and coding.  Let me fix it properly.

Reply:
Addressed in #25778 and make this PR depend on #25778.